apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -108,6 +114,29 @@ export const addStudentsFailure = (numStudents, error, studentIds) => (
 export const addMultipleRows = (studentData) => ({ type: ADD_MULTIPLE_ROWS, studentData });
 export const toggleSharingColumn = () => ({type: TOGGLE_SHARING_COLUMN});
 
+export const updateShareSetting = (sectionId, shareSetting) => dispatch => {
+  dispatch({type: UPDATE_SHARING_REQUEST});
+  return new Promise((resolve, reject) => {
+    $.ajax({
+      url: `dashboardapi/sections/${sectionId}/update_sharing_disabled`,
+      method: 'POST',
+      contentType: 'application/json;charset=UTF-8',
+      data: JSON.stringify({sharing_disabled: shareSetting}),
+    }).done(result => {
+      dispatch({
+        type: UPDATE_SHARING_SUCCESS,
```

Comment:
The part that's getting tricky is figuring out how  `UPDATE_SHARING_SUCCESS`, `UPDATE_SHARING_FAILURE` and `UPDATE_SHARING_REQUEST` fit in.  I've currently just copied them over from `teacherSectionsRedux`.  But the other ajax requests in `manageStudentsRedux` just do something like 
```  }).done((data) => {
    onComplete(null, data);
  }).fail((jqXhr, status) => {
    onComplete(status, null);
  });```

Also I think what we probably want to do is mirror something similar to success or failure of `EDIT_ALL` ?